src/sentry/tasks/statistical_detectors.py

```diff
@@ -933,11 +947,7 @@ def query_transactions(
         query=query,
         tenant_ids={
             "referrer": Referrer.STATISTICAL_DETECTORS_FETCH_TOP_TRANSACTION_NAMES.value,
-            # HACK: the allocation policy is going to reject this query unless there is an org_id
-            # passed in. The allocation policy will be updated to handle cross-org queries better
-            # As it is now (09-13-2023), this query will likely be throttled (i.e be slower) by the allocation
-            # policy as soon as we start scanning more than just the sentry org
-            "organization_id": -42069,
+            "cross_org_query": 1,
```

Comment:
This looks like it came from a bad merge?

Reply:
don't we still need a cross-org query here?

Reply:
I just replicated what we did for timeseries fetching

Reply:
Oh, I'm not actually sure. @volokluev can probably confirm if this is needed or not. If it is, feel free to ignore my comment.

Reply:
yes you do